src/core/parallelization_technique/src/ParallelizationTechnique.cpp

```diff
@@ -1811,6 +1811,22 @@ Value *ParallelizationTechnique::fetchCloneInTask(Task *t, Value *original) {
   return iClone;
 }
 
+std::string ParallelizationTechnique::
+    parallelReplacementNameForInputOutputFunction(
+        const std::string &originalName,
+        bool hasKnownMaxPrintedBytes) {
+
+  int unlocked = originalName.rfind("_unlocked");
```

Comment:
Can you use "auto" here?

Reply:
Fixed (and made the logic more clear, previously `unlockedSuffixIdx` would always be true and substr would always be used (which was fine because `npos` is the highest possible value, but this is how it was meant to work)).